react/postalCodeAutoCompleteAddress.test.js

```diff
@@ -102,25 +101,27 @@ describe('postalCodeAutoCompleteAddress()', () => {
     )
   })
 
-  it('should keep address id when auto completing fields', done => {
+  it('should keep address id when auto completing fields', async () => {
     const addressId = 'addressId1autoComplete'
     const modifiedAddress = {
       ...address,
       addressId: { value: addressId },
       postalCode: { value: '22251000' },
     }
 
-    function callback(data) {
-      expect(data.addressId.value).toBe(addressId)
-      done()
-    }
-
-    postalCodeAutoCompleteAddress({
-      cors,
-      accountName,
-      address: modifiedAddress,
-      rules: usePostalCode,
-      callback,
+    await new Promise((resolve) => {
+      function callback(data) {
+        expect(data.addressId.value).toBe(addressId)
+        resolve()
```

Comment:
`resolve()` is raising an error

```
Expected 1 arguments, but got 0. Did you forget to include 'void' in your type argument to 'Promise'?ts(2794)
lib.es2015.promise.d.ts(33, 34): An argument for 'value' was not provided.
```

Changing the file to `.ts` and making the promise return `void` works for me

```suggestion
    await new Promise<void>((resolve) => {
      function callback(data) {
        expect(data.addressId.value).toBe(addressId)
        resolve()
```

Reply:
how did you saw this error? it doesn't seem to block the link/build

Reply:
Weird, it popped on my IDE

![image](https://user-images.githubusercontent.com/26108090/124144481-045c4600-da5a-11eb-9dde-3a95e8e859a7.png)